content/data-sourcing.md

```diff
@@ -11,33 +11,33 @@ nav_order: 4
 
 The major challenge remaining for ASmap is to source and aggregate data about which AS owns which IP address range. Several entities, public and private, make BGP announcement data available. The following data sources represent the current state of research, and all remain incomplete or problematic in some way.
 
-### RIPE RIS
+### RPKI
 
-RIPE RIS has been used as the source for all ASMaps so far, but the project describes itself the data as "useful for looking at the state of the BGP Internet, debugging/post-mortems of events in BGP, and tracking of long-term trends in BGP" [RIS docs](https://ris.ripe.net/docs/mrt/). As such, data is collected for research purposes to observe and examine malicious behavior after the fact and learn from it. This means RIS would go out of its way to include data such as BGP hijacks and fat-finger type of leaks. There is no filtering or clean-up (after a leak was identified) of this data. The RRC's location and functionality are also publicly known so sending malicious announcements with the goal of getting them into RIS is comparatively easy.
+We can download [RPKI](http://localhost:4000/glossary/#rpki-resource-public-key-infrastructure) [ROAs](http://localhost:4000/glossary/#roa-route-origin-authorization) and build a prefix to AS mapping from it that is validated by the trust chain from one of the RIRs. This gives us a much higher quality of data.
```

Comment:
Does this work with the localhost links when this is deployed? I think you need to use a relative path or possibly `{{ site.baseurl}}` but the relative path has worked well for me as far as I can remember.